orcid-core/src/main/java/org/orcid/core/manager/impl/ProfileEntityManagerImpl.java

```diff
@@ -82,7 +84,8 @@
     private JpaJaxbWorkAdapter jpaJaxbWorkAdapter;
 
     @Override
-    public ProfileEntity findByOrcid(String orcid) {
+    @Cacheable(value = "profile-entity", key = "#orcid.concat('-').concat(#lastModified)")
```

Comment:
Hey @rcpeters,

https://trello.com/c/T4VGtkzU/1702-change-the-way-groupprofile-is-loaded

Please check this, now, when we get the group info from the ClientDetailsEntity class, we just get an orcid string, then, we call this to get the ProfileEntity, which, hopefully will be already cached.

Thanks